docker/webui/haproxy-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+cp /usr/local/etc/haproxy/haproxy.cfg.template /usr/local/etc/haproxy/haproxy.cfg
+for i in $(seq 2 $replicas); do
+  echo "  server webui$i webui_webui_$i:9526" >>/usr/local/etc/haproxy/haproxy.cfg
+done
+
+/usr/local/sbin/haproxy -f /usr/local/etc/haproxy/haproxy.cfg
+
+# Infinite wait to prvent early exit
+while true; do /bin/sleep 10; done
```

Comment:
why does it have to be `/bin/sleep`? Does the bash internal sleep not work?

Reply:
Checked it worked. For some reason I had to put this with the full path, but is not needed any more